### PR TITLE
pycon-au-2015

### PR DESCRIPTION
**Pull request by logston**

### CODE REVIEW

File: data/pycon-au-2015/videos/lightning-talks-19.json [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-  "language": "English", 
-  "copyright_text": "creativeCommon", 
-  "thumbnail_url": "https://i.ytimg.com/vi/MIRTtmjAKEw/hqdefault.jpg", 
-  "duration": null, 
+  "id": 3969,
+  "category": "PyCon AU 2015",
+  "slug": "lightning-talks-19",
+  "title": "Lightning talks",
+  "summary": "",
+  "description": "00:00 Introduction 01:10 Marcus Holterman - Django Logging Canary 02:54\nTim Ansell - Tim (still) has too many projects 08:24 Russell\nKeith-Maggee - I am a doctor 12:51 Steven Joeseph - Async IO TLDR\nedition 20:20 Chris Beevan - Bit ID Authentication 25:21 Xuanyi Chew -\nMore stuff you probably didn't know about 27:18 Aimee Marie Forsstrom -\nCyber Terrorism or Terrible Code 31:42 Rhys Elsmore - What Postgres can\nteach us about JavaScript 34:38 Roman Juce - zc.buildout for repeatible\nbuilds 38:10 Quentin Lovett - Building Robots 41:06 Amber 'Hawkie' Brown\n- Just use PyPy\n",
```
Comment (redapple) [inline data/pycon-au-2015/videos/lightning-talks-19.json:7]: Newlines are lost, making the description less readable:

```
+-----------------+---------+--------------------------------------------------------------------------+----------+
| Title           | Summary | Description                                                              | Speakers |
+-----------------+---------+--------------------------------------------------------------------------+----------+
| Lightning talks |         | 00:00 Introduction 01:10 Marcus Holterman - Django Logging Canary 02:54  | []       |
|                 |         | Tim Ansell - Tim (still) has too many projects 08:24 Russell             |          |
|                 |         | Keith-Maggee - I am a doctor 12:51 Steven Joeseph - Async IO TLDR        |          |
|                 |         | edition 20:20 Chris Beevan - Bit ID Authentication 25:21 Xuanyi Chew -   |          |
|                 |         | More stuff you probably didn't know about 27:18 Aimee Marie Forsstrom -  |          |
|                 |         | Cyber Terrorism or Terrible Code 31:42 Rhys Elsmore - What Postgres can  |          |
|                 |         | teach us about JavaScript 34:38 Roman Juce - zc.buildout for repeatible  |          |
|                 |         | builds 38:10 Quentin Lovett - Building Robots 41:06 Amber 'Hawkie' Brown |          |
|                 |         | - Just use PyPy                                                          |          |
|                 |         |                                                                          |          |
+-----------------+---------+--------------------------------------------------------------------------+----------+

```

vs.

```
+-----------------+---------+------------------------------------------------------------------+----------+
| Title           | Summary | Description                                                      | Speakers |
+-----------------+---------+------------------------------------------------------------------+----------+
| Lightning talks |         | 00:00 Introduction                                               | []       |
|                 |         | 01:10 Marcus Holterman - Django Logging Canary                   |          |
|                 |         | 02:54 Tim Ansell - Tim (still) has too many projects             |          |
|                 |         | 08:24 Russell Keith-Maggee - I am a doctor                       |          |
|                 |         | 12:51 Steven Joeseph - Async IO TLDR edition                     |          |
|                 |         | 20:20 Chris Beevan - Bit ID Authentication                       |          |
|                 |         | 25:21 Xuanyi Chew - More stuff you probably didn't know about    |          |
|                 |         | 27:18 Aimee Marie Forsstrom - Cyber Terrorism or Terrible Code   |          |
|                 |         | 31:42 Rhys Elsmore - What Postgres can teach us about JavaScript |          |
|                 |         | 34:38 Roman Juce - zc.buildout for repeatible builds             |          |
|                 |         | 38:10 Quentin Lovett - Building Robots                           |          |
|                 |         | 41:06 Amber 'Hawkie' Brown - Just use PyPy                       |          |
|                 |         |                                                                  |          |
|                 |         |                                                                  |          |
+-----------------+---------+------------------------------------------------------------------+----------+
```

File: data/pycon-au-2015/videos/lightning-talks-closing.json [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
-  "language": "English", 
-  "copyright_text": "creativeCommon", 
-  "thumbnail_url": "https://i.ytimg.com/vi/_pfJZfdwkgI/hqdefault.jpg", 
-  "duration": null, 
+  "id": 3986,
+  "category": "PyCon AU 2015",
+  "slug": "lightning-talks-closing",
+  "title": "Lightning Talks & Closing",
+  "summary": "",
+  "description": "00:00 Intro 00:25 Nic Couglan - Asyncio and native co-routines in python\n3.5 05:35 Tyson Clugg - Selenium Page Adaptor 10:25 Tom Eastman -\nSerialisation formats are not toys 16:00 Ben Davies - My Eyes, the\ngoggles do nothing 20:43 Paula Andrea Martinez - Software Carpentry\n24:17 Alex Hill - Django Many to Many Minus the Middle Men 29:34 Adam\nBreniki - Who remembers VisiCalc 33:34 Fraser Tweedale - Fast and Loose\nReasoning and Parametricity 38:47 Amber 'Hawkie' Brown - Pythonic Naming\n42:20 Sam Bishop - Distraction 47:36 Tim Bell - Time Travel Debugger\n52:29 Brian Thron - Roll your own crypto 57:18 Thuran Sunti - UQ\nRobotics Club 1:02:35 Robert Collins - Data driven testing\n",
```
Comment (redapple) [inline data/pycon-au-2015/videos/lightning-talks-closing.json:7]: Newlines are lost. Same as for `data/pycon-au-2015/videos/lightning-talks-19.json`

**Comment by redapple** [comment]
Apart from the schedule for 2 lightning talks sessions which loose formatting, looks good to me.

**Comment by willkg** [comment]
I was thinking about what to do with lightning talks. I don't like what we're doing now (which I think I started a few years back). One of the problems is that it's encoded in formatting and isn't easy to extract.

I think maybe we should add the idea of "chapters" to the schema or something like that. I wrote up #108 for that.

I'm game for landing this and then I can go fix the newline issues. It's probably the case that it's going to be a problem for any of the lightning talks where I formatted things the way I did across all the videos. :(

Thoughts?

**Pull request merged**